### PR TITLE
Eli/cer 3047 cli reworks and bugfixes

### PR DESCRIPTION
**Pull request by elijah-rou**

### CODE REVIEW

File: cerebrium/getting-started/quickstart.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: "Get up and running with your first deployed model on Cerebrium"
 The fastest way to get started developing a Cerebrium deployment is to set up a template project using the `cerebrium init` command below. This will create a folder with all the necessary files to get you started. You can then add your code and deploy it to Cerebrium.
 
 ```bash
-cerebrium init first-project
+cerebrium init first-project --name=cerebrium-app
```
Comment (jonoirwinrsa) [inline cerebrium/getting-started/quickstart.mdx:9]: what is "first-project" if not the name?

**Review by jonoirwinrsa — APPROVED** [review]
just the 1

File: cerebrium/getting-started/quickstart.mdx (hunk 1 above)
Comment (elijah-rou) [reply cerebrium/getting-started/quickstart.mdx:9]: the directory

Comment (elijah-rou) [reply cerebrium/getting-started/quickstart.mdx:9]: been like that for a while

**Pull request merged**